packages/devtools/src/types.ts

```diff
@@ -1,9 +1,20 @@
-import type { Capabilities } from '@wdio/types'
+import type { Options, Capabilities } from '@wdio/types'
 
 export interface ExtendedCapabilities extends Capabilities.Capabilities {
     'wdio:devtoolsOptions'?: DevToolsOptions
 }
 
+export interface SessionFlags {
```

Comment:
This has been already defined in the `webdriver` package. If we need it here too it should be moved into `@wdio/types` to avoid code duplication.